R/regressiondiagnostics.R

```diff
@@ -361,22 +361,22 @@ issvyglm <- function(model)
 
 #' @importFrom effects allEffects
 #' @export
-allEffects.Regression <- function(model, ...)
+allEffects.Regression <- function(mod, ...)
```

Comment:
just curious why rename model to mod?

Reply:
The generic uses `mod` as the argument name. The name `model` is used here which is a mismatch and causes a warning in the `R CMD check`. Any warnings now cause a build fail on CircleCI.
https://app.travis-ci.com/github/Displayr/flipRegression/builds/262249901 Example here.